app/main.py

```
@@ -63,65 +63,89 @@ def get_log_level():
+    Queries TeamCity for projects marked as archived and returns their IDs.
     
     Returns:
-        list[str]: Project IDs marked as archived (empty list if no archived projects).
+        list[str]: List of archived project IDs (empty list if none).
     """
     logging.debug("Reached get_archived_projects")
-    url = f"{TEAMCITY_URL}/app/rest/projects?locator=archived:true"
-    resp = requests.get(url, headers=HEADERS)
-    resp.raise_for_status()
-    data = resp.json()
+    locator = f"archived:true"
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)